modules/manage/partials/azure-blob-limitations.adoc

```diff
@@ -1 +1 @@
-CAUTION: Redpanda requires a container in a dedicated storage account, and doesn't support storage accounts that have https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-namespace[hierarchical namespaces^] enabled.
+NOTE: Starting in release 23.2.8, Redpanda supports storage accounts configured for Azure Data Lake Storage (ADLS) with https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-namespace[hierarchical namespaces^]. For hierarchical namespaces created with a custom endpoint, set `cloud_storage_azure_adls_endpoint` and `cloud_storage_azure_adls_port`. If you haven't configured custom endpoints in Azure, there's no need to edit these properties.
```

Comment:
I don't think mentioning ADLS here is right. The previous version mentioned hierarchical namespaces which is the correct term.

Reply:
it says 'storage accounts configured for ADLS **_with_** hierarchical namespaces' ('with' being the operative word)

What would you say?  The field understands this a Gen1 container vs a Gen2 container